design/ceph/ceph-mon-pv.md

```diff
@@ -136,3 +136,132 @@ relies on determining if monitor pods may be placed on a node, requiring
 knowledge of the scheduling policy and algorithm. The scenario to watch out for
 is an endless loop in which the health check is determining a bad placement but
 the k8s schedule thinks otherwise.
+
+## Monitor Storage Class Support
+
+It is generally desirable in production to have multiple zones availability. A
+whole storage class could go unavailable then and enough mons could stay up to
+keep the cluster running.
+In such a case the distribution of mons can be made more homogeneous for each
+zone, by this we can leverage the spread of mons for providing high
+availability. This could be a useful scenario for a datacenter with own storage
+class in contrast to ones provided by cloud providers.
+
+Rook will enable the mon deployment when requested by the admin through the
```

Comment:
The section below seems like it already shows how the CRD works, so I think we can remove this sentence.
```suggestion
```